specification/ambulance-analytics.yaml

```diff
@@ -97,28 +101,13 @@ paths:
                 properties:
                   message:
                     type: string
-                    example: "Hello world!"
+                    example: "Test"
               example:
-                message: "Hello world!"
+                message: "{}"
       tags:
-        - hello
+        - Post ambulance data
 
 components:
-
-  securitySchemes:
-    api_key:
-      type: apiKey
-      name: apikey
-      in: header
-
-    oAuth2AuthCode:
-      type: oauth2
-      flows:
-        authorizationCode:
-          authorizationUrl: https://sandbox.api.service.nhs.uk/oauth2/authorize
-          tokenUrl: https://sandbox.api.service.nhs.uk/oauth2/token
-          scopes: {}
-
   schemas:
```

Comment:
I think this schema (Greeting) is unused. Or should be renamed

Reply:
Yeah I think it needs renaming and changing to the schema of the ambulance endpoint